romancal/ramp_fitting/tests/test_ramp_fit.py

```diff
@@ -128,7 +128,7 @@ def test_one_group_small_buffer_fit_ols(max_cores):
     data = out_model.data.value
 
     # Index changes due to trimming of reference pixels
-    np.testing.assert_allclose(data[11, 6], 10.0, 1e-6)
+    np.testing.assert_allclose(data[11, 6], -1.0e-5, 1e-6)
```

Comment:
To satisfy my curiosity, would you explain why this is the new expected value? Also, is ~10% tolerance reasonable?

Reply:
Frankly I'm not too worried about it. We're replacing this with a new method of fitting unevenly spaced ramps that is appropriate for Roman data. But from what I saw it is interpolating a value to replace the input cosmic ray and is consistent with the surrounding values.